erpnext/stock/doctype/batch/batch.py

```diff
@@ -326,7 +326,11 @@ def save_thc_cbd(batch_no, thc, cbd):
 	doc.save()
 
 @frappe.whitelist()
-def get_active_batch(item_code):
-	active_batch = frappe.get_all("Batch", {"item": item_code, "display_on_website": 1})
+def get_active_batch(item_code, coa=False):
+	if(coa):
+		fields = ["name", "certificate_of_analysis"]
+	else:
+		fields = ["name"]
+	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1},  fields=fields )
```

Comment:
You might want to get more fields then

```suggestion
def get_active_batch(item_code):
	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1},  fields=["name", "item", "item_name", "stock_uom", "thc", "cbd", "certificate_of_analysis"])
```

Reply:
Done.